warehouse/upload.go

```diff
@@ -2077,3 +2083,42 @@ func (job *UploadJobT) GetLocalSchema() warehouseutils.SchemaT {
 func (job *UploadJobT) UpdateLocalSchema(schema warehouseutils.SchemaT) error {
 	return job.schemaHandle.updateLocalSchema(schema)
 }
+
+func (job *UploadJobT) refreshPartitions(loadFileStartID, loadFileEndID int64) error {
+	if slices.Contains(warehouseutils.TimeWindowDestinations, job.upload.DestinationType) {
+		return nil
```

Comment:
s3_datalake also come under TimeWindowDestinations. refreshpartitions would be skipped

Reply:
Yeah, This should be negation.

Reply:
Will make the changes.